package.json

```diff
@@ -12,27 +12,33 @@
     "start": "stripes serve",
     "build": "stripes build --output ./dist",
     "test-int": "stripes test nightmare --run demo --show",
-    "test": "stripes test karma",
+    "test": "jest --coverage",
```

Comment:
Don't you think it will be good idea to create 2 commands for running test with and without coverage?
It's not necessary, just an idea.
```
"test": "jest",
"test:coverage": "jest --coverage",
```

Reply:
In my opinion this module is not so huge that we could omit coverage setting for better performance. We discuss this point with Denis and we could move this option to `Jenkinsfile` and only show coverage on CI, but it will be better to check coverage localy before creating the PR.